source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -292,141 +390,278 @@ Let’s take a look at the intermediate layer of our project to understand the p
+- ✅ **Build on separate marts thoughtfully.** While we strive to preserve a narrowing DAG up to the marts layer, once here things may start to get a little less strict. A common example is passing information between marts at different grains, for example, to aggregate data into the grain of another mart. Now that we’re really ‘spending’ compute and storage by actually building the data in our outputs, it’s sensible to leverage previously built resources to speed up and save costs on outputs that require similar data, versus recomputing the same views and CTEs from scratch. The right approach here is heavily dependent on your unique DAG, models, and goals — it’s just important to note that using a mart in building another, later mart is okay, but requires careful consideration to avoid wasted resources or circular dependencies.
+
+### Marts: Other considerations
 
-Every pull request to merge a branch with the main branch in create-a-derived-table requires a review by someone in the analytics engineering team and in some cases, when the changes affect project files, like `dbt_project.yml`, then a data engineer's reveiew is also needed. These reviews are to ensure that the code entering the codebase is inline with best practice and our guidance, as well as ensuring it wont disrupt the scheduled runs.
+- **Troubleshoot via tables.** While stacking views and ephemeral models up until our marts — only building data into the warehouse at the end of a chain when we have the models we really want end users to work with — is ideal in production, it can present some difficulties in development. Particularly, certain errors may seem to be surfacing in our later models that actually stem from much earlier dependencies in our model chain (ancestor models in our DAG that are built before the model throws the errors). If you’re having trouble pinning down where or what a database error is telling you, it can be helpful to temporarily build a specific chain of models as tables so that the warehouse will throw the error where it’s actually occurring.
 
-This guide will provide details on what exactly will be checked and what you need to do to ensure you project passes these checks. Below is the checklist we use to review any pull request
+## 5-derived
```

Comment:
Some notes from the discussion we had on the Derived layer on Tuesday:

- This layer essentially represents any datasets downstream of marts, usually built to answer a specific business need by flattening dimensions and facts into one or few tables.
- We proposed having two types of Derived dataset within this layer, distinguished by the database they belong to:
  - **AE-managed databases** named after relevant domains / sub-domains, e.g. 'probation' / 'finance' [unsure if we should have a suffix here instead, like 'probation_derived']. All tables within these databases have either been built by AEs or developed in consultation with AEs, and represent a gold standard table which is considered useful to many analysts working in that domain.
  - Any other **analyst-created databases** which, while passing the requisite review to be merged into production in CaDeT, are either too niche or unverified to be included among the AE-managed set of tables.
- Access to databases in the Derived layer will likely need to be granted on a database level, with relevant config in Database Access set up.
  - The current domain-level permissions structure could be amended so that instead of the resource 'create_a_derived_table/domain' granting access to every single model in 'domain', it could point specifically to the AE-managed databases instead, i.e. marts and derived.
  - More sensitive or team-specific derived tables would be separate resources only accessible to those who need them.